src/Protocol/Protocol_1_9.cpp

```diff
@@ -679,6 +679,7 @@ UInt32 cProtocol_1_9_0::GetPacketID(cProtocol::ePacketType a_Packet)
 		case pktUpdateScore:           return 0x42;
 		case pktUpdateSign:            return 0x46;
 		case pktUseBed:                return 0x2f;
+		case pktUnlockRecipe:          return 0x31;
```

Comment:
This doesn't belong in `cProtocol_1_9_0`

Reply:
Well, I have to add it, otherwise the CI server complains about:
```
/Users/travis/build/cuberite/cuberite/src/Protocol/Protocol_1_9.cpp:498:10: error: enumeration value 'pktUnlockRecipe' not handled in switch [-Werror,-Wswitch]
        switch (a_Packet)
```
Or is there any other solution?

Reply:
Ok, looked a bit more into it, but I couldn't find a better way.

If I understand it correctly I have to define the `pktUnlockRecipe` in the `Protocol.h`. When defining it there at least `clang` complains if it is not covered in the switch statement in the `Protocol_x_x.cpp`. So I need to define it in 1.8 and 1.9, too.

Am I missing something here?